Windows/Build/Deployment/azure-pipelines.yml

```diff
@@ -0,0 +1,92 @@
+trigger:
+  batch: true
+  branches:
+    include:
+      - master
+  paths:
+    include:
+      - "Windows/Build/Deployment"
+    exclude:
+      - "*"
+
+pr:
+  paths:
+    include:
+      - "Windows/Build/Deployment"
+
+pool:
+  name: DAS - Continuous Integration Agents
+
+resources:
+  repositories:
+  - repository: das-platform-building-blocks
+    type: github
+    name: SkillsFundingAgency/das-platform-building-blocks
+    ref: refs/tags/0.4.63
+    endpoint: SkillsFundingAgency
+  pipelines:
+  - pipeline: image-build
+    source: das-azure-pipelines-agents-win-build (Image Build)
+    branch: master
+    trigger:
+      branches:
+        - master
+
+variables:
+- group: RELEASE das-azure-pipelines-agent
+- group: Release Management Resources
+- name: ContainerImageName
+  value: azure-pipelines-build-agent-win
+- name: ImageTag
+  value: $(resources.pipeline.image-build.runName)
+
+stages:
+  - stage: DeployDev
+    variables:
+    - group: DEV Windows Build Agent
+    - group: Dev Management Resources
+    jobs:
+    - job: DeployManifest
+      displayName: DeployManifest
+      steps:
+        - template: /pipeline-templates/step/deploy.yml
+          parameters:
+            VerifyDeployment: true
+            Directory: $(System.DefaultWorkingDirectory)/Windows/Build/Deployment
+            AzureServiceConnection: SFA-DAS-DevTest-ARM
+        - pwsh: |
+            Start-Sleep -Seconds 120 # Do Better. This gives time for the old pods to remove themselves from the pool before AcceptanceTests.
+
+  - stage: AcceptanceTests
+    dependsOn: DeployDev
+    jobs:
+    - job: DotNetAcceptanceTests
+      pool:
+        name: DAS - Continuous Integration Beta
+        demands:
+          - Agent.OS -equals Windows_NT
+      displayName: RunTests
+      strategy:
+        matrix:
+          DotNet3.1:
```

Comment:
Can we test for .NET Framework as well?

Reply:
dotnet new supports from core 3.0: https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-new-sdk-templates#:~:text=Specifies%20the%20framework%20to%20target.%20Available%20since%20.NET%20Core%203.0%20SDK.
Would be a nice to have in the future, but even better would be to get rid of it in our apps😁